model/event/host_event.go

```diff
@@ -238,10 +238,6 @@ func LogHostRunningTaskCleared(hostId string, taskId string) {
 		HostEventData{TaskId: taskId})
 }
 
-func LogHostTaskPidSet(hostId string, taskPid string) {
```

Comment:
https://github.com/evergreen-ci/evergreen/blob/main/model/event/host_event.go#L53
Is this constant still needed?

Reply:
no! removed